commons/src/main/java/com/powsybl/commons/report/ReportNodeRootBuilderImpl.java

```diff
@@ -42,7 +42,14 @@ public ReportNode build() {
     }
 
     @Override
-    public ReportNodeBuilder self() {
+    public ReportNodeRootBuilder self() {
         return this;
     }
+
+    @Override
+    public ReportNodeRootBuilder withName(String name) {
+        this.key = "core.report.name";
+        values.put("name", TypedValue.untyped(name));
```

Comment:
It may be better to use a less generic name for the value (for instance `name` -> `reportName`), since it will be inherited (and therefore usable) by the child nodes.